RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,34 @@
+# Release Checklist
+
+This is a list of the things that need to happen during a release.
+
+1. Open the associated milestone. All issues and PRs should be closed. If
+  they are not you should reassign all open issues and PRs to future
+  milestones.
+1. Go through the commit history since last release. Ensure that all PRs
+  that have landed are marked with the milestone.
+1. Go through the closed PRs in the milestone. Each should have a changelog
+  label indicating if the change is docs, fix, feature, or maintenance. If
+  there is a missing label, please add one.
+1. Choose an emoji for the release. Try to make it some sort of transition
```

Comment:
I'm not sure I understand the transition part, could you elaborate this?

Reply:
so i try to tell a small story as the emoji per release progress, e.g. we went from 🌌 to 💥 sorta like... the Big Bang. and then transitioning through the stars to the sun to sun with clouds... it's not a formality but i think it's a fun little way to make chosing the release emoji a little more constrained. it's certainly not something to block on.

Reply:
Alright, that seems like a fun way of doing it :blush: